### DISCUSSION TITLE
Can we have better CSS by default?

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#486] [Ideas]
NGL but the default css for the link hints and the vomnibar borrowed from vimium is pretty ugly. I'm sure we can have visually much better css without any bloat.

**Comment by gdh1995** [comment]
I want to make Vimium C use "as few as possible" CSS rules to show its hints, to save CPU. It's mainly because, Vimium C's `LinkHints` has considered about so many cases that it runs a bit slower than Vimium - then, other parts should be faster if only possible.

**Comment by txtyash** [comment]
But we can definitely choose much better colors. I'm not referring to more/better CSS. I'm talking about the default color choices. 

The ugly yellow hint markers with plain white text -> material yellow hint markers with material gray text and dark yellow borders.

**Comment by gdh1995** [comment]
Not slow, but I still don't like to change the "default styles", in order to ensure old users' experience doesn't change.

And in fact I like the orange-like gradients. As for Vomnibar, white/background colors make it clear to read.

**Reply by txtyash** [reply]
Yes, vomnibar seems fine but the hints don't. And about that consistent experience. I think that people who don't follow the development would notice that the plugin is maintained well and improvements are being made. Also adds up to a small purpose of choosing vimiumC over vimium as no one would be able to telll apart vimimC and vimium as they clearly won't be able to tell which one would be a better choice because vimium has more rating and both look same by default.

**Reply by gdh1995** [reply]
Vimium C is designed as an "improved" version of Vimium (at least in most places), and I think "keeping Vimium-compatible" is a way good enough to attract Vimium's users - as you can see, now Vimium's development is in very very slow, so there will always be some users expecting new features.
